cmd/zpool/zpool_main.c

```diff
@@ -7619,7 +7619,7 @@ print_scan_scrub_resilver_status(pool_scan_stat_t *ps)
 
 	if (pause == 0) {
 		if (total_secs_left != UINT64_MAX &&
-		    issue_rate >= 10 * 1024 * 1024) {
+		    issue_rate >= 10 * 1024 * 1024 && ps->pss_processed > 0) {
```

Comment:
During scrub on a draid2 pool, the scan field shows "no estimated completion time" and only after the scrub is complete the time is printed in the scan field. I'm not sure if this is due to the size of the pool or the drives used.
Reverting this line gives me back the scan field with the estimated time.

Reply:
That's no good, my intent here was to only report an estimate once we started issuing repair I/O and thus could make a reasonable assessment.   But as you noticed that clearly doesn't work for a scrub where we don't expect any repair to be needed.  I went ahead and updated the check to only require that I/O be issued for the scrub case.